### PR TITLE
New concepts part 1

### PR DESCRIPTION
**Pull request by CarolinHu**
# Pull request checklist

- [ ] add new concept list
- [ ] add new metadata
- [x] add new Concepticon concept sets
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

First half of the concepts proposed in issue #850 plus mappings in five lists. If you find more NAs that fit one of the new concepts, let me know!
I saw that Bowern-2012-644 is also a Lexibank repos, so if there's no disagreement about the mapping done here, I will open an issue over there.

@chrzyki, would you mind being moderator for this PR?

### CODE REVIEW

**Comment by chrzyki** [comment]
Heya,

I'm on holiday this week (until Tuesday next week) so I might be a bit slower to respond.

Concerning additional mappings:

- Izura-2005-499-404 OSTRICH
- DiezAlamo-2018-750-20 OSTRICH
- Borin-2015-1532-713 STINGER (maybe, please check)

@Kristina-Pianykh & @AnnikaTjuka, would you mind checking this as well? Thanks!

**Comment by CarolinHu** [comment]
Sorry about this. Please feel free to not be the moderator and enjoy your holidays instead! I can ask someone else. (Will implement the suggested mappings, though. Thanks for the Spanish gloss. The Borin concept "the scorpion", however, is mapped to SCORPION, which definitely seems the most fitting. Is there something I am missing here?)

**Comment by chrzyki** [comment]
Ah, sorry, misread the Borin gloss! That can be ignored then. Re moderating: It's fine, no worries!

**Comment by AnnikaTjuka** [comment]
I'll submit my review tomorrow!

File: concepticondata/concepticon.tsv [hunk 1]
```diff
@@ -3779,3 +3779,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3793	EIGHT THOUSAND	Quantity	The natural number eight thousand (8000).	Number	
 3794	TWO MILLION	Quantity	The natural number two million (2000000).	Number	
 3795	DARK GREEN	Sense perception	A darker variant of the color green, mixed with black, with an approximate wavelength between 543 and 549 nm.	Property	
+3796	ALBATROSS	Animals	Any seabird of the family Diomedeidae. Some species are among the largest flying birds in the world.	Person/Thing	
+3797	OSTRICH	Animals	Any of two species of large, flightless birds in the family Struthionidae, native to Africa. 	Person/Thing
```
Comment (AnnikaTjuka) [inline concepticondata/concepticon.tsv:3783]: I'd add "of" in "in the family of Struthionidae".

Comment (SimonGreenhill) [reply concepticondata/concepticon.tsv:3783]: no, that sounds weird :) -- biologists say "family X"

Comment (AnnikaTjuka) [reply concepticondata/concepticon.tsv:3783]: Oh ok, good to know :) We have a few definitions where "family of" is used, so I had this wording in mind.

Comment (SimonGreenhill) [reply concepticondata/concepticon.tsv:3783]: Ahh, then consistency is more important, so go with what everything else is.

**Comment by chrzyki** [comment]
Thanks, feel free to merge when ready.

File: concepticondata/concepticon.tsv (hunk 1 above)
Comment (CarolinHu) [reply concepticondata/concepticon.tsv:3783]: I just quickly checked some descriptions of birds that first came to mind - parrot, eagle, sparrow, hummingbird - and there, it is always "family" without "of". So, if that sounds more natural and both versions exist in the definitions, I'm going to leave it as it is. Thanks for reviewing!

**Pull request merged**